docs/rest/SMARTScopesExample.http

```diff
@@ -21,9 +21,36 @@ grant_type=client_credentials
 &client_secret=globalAdminServicePrincipal
 &scope=patient/Observation.read fhirUser fhir-api user/Encounter.*
 
+### GET
+https://localhost:44348/_operations/export/a6abdbf4-f5a6-4feb-a795-7c76cabcca69
```

Comment:
Small change: replace "localhost:44348" with {{hostname}}.

Reply:
Removed, as the GUID for the export job changes each time.  Probably shouldn't be in the file.